packages/ui/src/components/va-time-picker/VaTimePickerColumn/VaTimePickerColumn.vue

```diff
@@ -139,6 +157,14 @@ export default defineComponent({
     height: 100%;
     border-right: var(--va-time-picker-column-border-right);
 
+    &::before,
+    &::after {
+      content: "";
+      display: block;
+      height: var(--va-time-picker-column-gap-height);
```

Comment:
I think we need to calculate component height (`--va-time-picker-height: 200px;`) and gap (`--va-time-picker-column-gap-height: 85px;`) based on cell height (`--va-time-picker-cell-height`)